README.md

```diff
@@ -69,11 +69,37 @@ const options = {
   logger                           // [optional] logger function to be called with errors
 };
 
-terminus(server, options);
+createTerminus(server, options);
 
 server.listen(PORT || 3000);
 ```
 
+### With custom error messages
+
+```js
+const http = require('http');
+const { createTerminus, TerminusError } = require('@godaddy/terminus');
+
+createTerminus(server, {
+  healthChecks: {
+    '/healthcheck': async function () {
+      const errors = []
+      return Promise.all([
+        // all your healthchecks goes here
+      ].map(p => p.catch((error) => {
+        // silently collecting all the errors
+        errors.push(error)
+        return undefined
+      }))).then(() => {
+        if (errors.length) {
+          throw new TerminusError('healtcheck failed', errors)
```

Comment:
The name `TerminusError` may cause problems. Imagine in the future you throw an error in another context e.g. Signals. Then you should have a different error type per context, otherwise the user can not distinguish the error type. Therefore I suggest a `HealthCheckError` or however you call it internally, which inherits `TerminusError` which inherits `Error`.

It is just a minor thing, but I think it could prevent major changes in the future.